assets/js/sidebar.js

```diff
@@ -49,7 +62,9 @@ function openSidebar (immediate) {
     '-ms-flex-basis': width,
     'flex-basis': width,
     width: width
-  }, duration)
+  }, duration, function () {
+    sidebar.attr('style', '')
```

Comment:
I have no idea. I would love if you could help me find out. The bug report is here: #505 and you can reproduce by just making the window small.